app/controllers/users_controller.rb

```diff
@@ -5,6 +5,6 @@ def index
 
   def show
     @user = User.find(params[:id])
-    @recent_posts = @user.recent_posts
+    # @recent_posts = @user.recent_posts
```

Comment:
- [ ] (OPTIONAL) Kindly remove all commented code so as to keep your work clean :100: 
- [ ] (OPTIONAL) It would be great if you could make the `# root "users#index"` as your default route so that your app takes the user directly to your home page instead of the rails default page :+1: